x-pack/plugin/sql/qa/src/main/resources/docs/docs.csv-spec

```diff
@@ -2783,6 +2784,42 @@ schema::datetime:ts
 // end::dateTimeParse3
 ;
 
+timeParse1
+schema::time:time
+// tag::timeParse1
+SELECT TIME_PARSE('10:20:30.123', 'HH:mm:ss.SSS') AS "time";
+
+     time
+---------------
+10:20:30.123Z
+// end::timeParse1
+;
+
+timeParse2
+schema::time:time
+// tag::timeParse2
+SELECT TIME_PARSE('10:20:30-01:00', 'HH:mm:ssXXX') AS "time";
+
+     time
+---------------
+11:20:30.000Z
+// end::timeParse2
+;
+
+timeParse3-Ignore
+schema::time:time
+// tag::timeParse3
+{
+    "query" : "SELECT DATETIME_PARSE('10:20:30', 'HH:mm:ss') AS \"time\"",
+    "time_zone" : "Europe/Athens"
+}
+
+      time
+------------------------------------
+08:20:30.000Z
```

Comment:
```suggestion
10:20:30.000+03:00
```

Reply:
`LocalDateTime ldt = lt.atDate(LocalDate.of(1970, 1, 1));` I has one question. In 1970, the offset time of Athens is '+02:00',  but the offset time of Athens is '+03:00'. I think using `LocalDateTime.now().toLocalDateTime()` is correctly.

Reply:
Let's leave the code with LocalDate.EPOCH and use `+02:00` in the example.

Reply:
Do I need to explain it in notes？

Reply:
Yes please! thank you.

Reply:
Let me try. I'm not good at English.